### ISSUE TITLE
Unexpected Error related to urllib3

### ISSUE THREAD

**Issue by rojas70** [conda/conda#5039]
Using Anaconda2-4.3.1-Linux-x86_64 in Linux 14.04.5 with Python 2.7. Anytime I use any conda command I get the following error:

````
>>$ conda -h
An unexpected error has occurred.
Please consider posting the following information to the
conda GitHub issue tracker at:

    https://github.com/conda/conda/issues



Traceback (most recent call last):
  File "/home/vmrguser/anaconda2/bin/conda", line 6, in <module>
    sys.exit(conda.cli.main())
  File "/home/vmrguser/anaconda2/lib/python2.7/site-packages/conda/cli/main.py", line 164, in main
    return conda_exception_handler(_main, *args)
  File "/home/vmrguser/anaconda2/lib/python2.7/site-packages/conda/exceptions.py", line 589, in conda_exception_handler
    print_unexpected_error_message(e)
  File "/home/vmrguser/anaconda2/lib/python2.7/site-packages/conda/exceptions.py", line 543, in print_unexpected_error_message
    info_stdout, info_stderr = get_info()
  File "/home/vmrguser/anaconda2/lib/python2.7/site-packages/conda/exceptions.py", line 513, in get_info
    args.func(args, p)
  File "/home/vmrguser/anaconda2/lib/python2.7/site-packages/conda/cli/main_info.py", line 148, in execute
    from conda.api import get_index
  File "/home/vmrguser/anaconda2/lib/python2.7/site-packages/conda/api.py", line 1, in <module>
    from .core.index import get_index
  File "/home/vmrguser/anaconda2/lib/python2.7/site-packages/conda/core/index.py", line 9, in <module>
    from .repodata import collect_all_repodata
  File "/home/vmrguser/anaconda2/lib/python2.7/site-packages/conda/core/repodata.py", line 21, in <module>
    from requests.packages.urllib3.exceptions import InsecureRequestWarning
ImportError: No module named packages.urllib3.exceptions
````

I have already tried to install/upgrade urllib3, requests, and boxsdk with pip, but to no avail. Anyone know how to fix this?

**Comment by wsAndy** [comment]
I download Anaconda(python 2.7 & ubuntu 64 bit ) and has the same problem.

**Comment by wsAndy** [comment]
I use Anaconda2-4.1.1-Linux-x86_64 in ubuntu 14.04 64 bit , and this problem not occurs

**Comment by kalefranz** [comment]
fixed with #5090

**Comment by seaun163** [comment]
I have the same problem.Because of I have two versions of python,So I fixed it by  change my python's version. Just edit:"gedit ~/.bashrc"
